extension/resources/walkthrough/plots.md

```diff
@@ -1,61 +1,72 @@
-# Plots
+# Plots Dashboard
 
-Select one or more experiments to visualize them in the
-[`Plots Dashboard`](command:dvc.showPlots). This is the extension's equivalent
-of the `dvc plots show` and `dvc plots diff` commands.
+Select one or more experiments to visualize in the
+[**Plots Dashboard**](command:dvc.showPlots). Use
+[`DVC: Show Plots`](command:workbench.action.quickOpen?%22>DVC:%20Show%20Plots%22)
```

Comment:
Do we need both links? I think they do the same thing in the end.

Reply:
it educates ppl a bit more